spec/factories/set_member_subjects.rb

```diff
@@ -2,10 +2,7 @@
   factory :set_member_subject do
     subject_set
     subject
-
-    trait :with_priorities do
-      sequence(:priority)
-    end
+    sequence(:priority)
```

Comment:
Mind using the trait `with_priorities` where you need it vs adding priorities to any sms factory record? Not sure what the flow on effects are for this change and the real world data will be mostly sms records without priority.

Reply:
RSpec doesn’t allow one to use a trait while also passing override values (subject_set_id, in this case) to `create_list`. I’m sure there’s another way to do it, but I figured that sequence wasn’t heavy. If those values being there could cause issues elsewhere I’ll remove them but that would also be good to know.

Reply:
TBH i'm not sure about repercussions. 

I'd try and keep the test data as real world as we can. most subjects / SMS won't have the priority data so best to keep them off the default factories i think